src/main/resources/db/changelog/v1_0_liquibase_create_users_schema.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                            http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet id="1660725967905" author="temitope">
+        <createTable tableName="users">
+
+            <column name="deleted" type="BIT"/>
```

Comment:
For better readability, maybe move the deleted, created on and last modified to the line 35 and let the id be the first